backend/controllers/orgUsersController.js

```diff
@@ -1,20 +1,58 @@
 const OrgUser = require("../models/orgUserModel");
-const mongoose = require('mongoose');
+const jwt = require("jsonwebtoken");
 
-const getAllOrgUsers = async (req, res) => {
```

Comment:
Can we leave this in for now for testing in Postman?

Reply:
Yes sorr

Reply:
Hmm...  main doesn't have getAllOrgUsers  - do you want me to readd it back?